components/prism-css-extras.js

```diff
@@ -35,7 +35,16 @@ Prism.languages.css.selector = {
 				],
 				'operator': /[|~*^$]?=/
 			}
-		}
+		},
+		'n-th': {
```

Comment:
`n-th` is a class in our themes already?

Reply:
No, but it's a necessary wrapper for the `number` and `operator` tokens.